packages/playground/README.md

```diff
@@ -64,5 +64,5 @@ More illustration on the build-env script [here](docs/build.md)
 ## Related Documentations
 
 - [Contributing Guide](./docs/contributing.md)
-- [playground Documentation](https://manual.grid.tf/weblets/weblets_home.html)
+- [Playground Documentation](https://www.manual.grid.tf/documentation/dashboard/deploy/applications.html)
```

Comment:
i think this one should be : https://www.manual.grid.tf/documentation/dashboard/dashboard.html

Reply:
If we change this link then we might as well change the playground introduction and description in the README as it is used for more than deploying solutions now. What do you think @xmonader ?

Reply:
I agree, the playground docs should be updated. but not necessarily in this PR